pkgs/applications/science/electronics/gerbv/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchgit, pkgconfig, gettext, libtool, automake, autoconf, cairo, gtk, desktop_file_utils}:
+
+stdenv.mkDerivation rec {
+  name = "gerbv-${version}";
+  version = "2015-10-08";
+
+  src = fetchgit {
+    url = git://git.geda-project.org/gerbv.git;
+    rev = "76b8b67bfa10823ce98f1c4c3b49a2afcadf7659";
+    sha256 = "1l2x8sb1c3gq00i71fdndkqwa7148mrranayafqw9pq63869l92w";
+  };
+
+  buildInputs = [ pkgconfig gettext libtool automake autoconf cairo gtk desktop_file_utils ];
+
+  preConfigure = ''
+    ./autogen.sh
+  '';
+
+#    postConfigure = ''
+# #    make
+#     sleep 5
+#     echo hey mog
+#     cd po
+#     msginit -l ru
+#     cd ..
+#     touch man/gerbv.ru.1.in.po
+#   '';
+
+  meta = with stdenv.lib; {
+    description = "Gerbv -- A Gerber (RS-274X) viewer";
+    homepage = http://gerbv.geda-project.org/;
+    maintainers = with maintainers; [ mog ];
+    platforms = platforms.linux;
+    license = licenses.gpl2;
+    priority = 10;
```

Comment:
when i built without that i got an error. in googling I saw someone setting that priority.  so i copy pasted.  don't know if its right thing to do